src/ruby.ts

```diff
@@ -192,7 +192,8 @@ export class Ruby {
 
     if (major < 3) {
       throw new Error(
-        "The Ruby LSP requires Ruby 3.0 or newer to run. Please upgrade your Ruby version",
+        "The Ruby LSP requires Ruby 3.0 or newer to run. \
```

Comment:
```suggestion
        "The Ruby LSP officially supports only Ruby 3.0 or newer. \
```

Reply:
I think I still prefer the `requires` here instead of `officially supports`. To me it sounds like there's unofficial support for older rubies, which isn't the case.

Reply:
Fair, but I think there still needs to be some subtlety in there to indicate that there is a way to work with older Rubies.